app/authorization/server/functions/canSendMessage.js

```diff
@@ -36,16 +36,6 @@ export const validateRoomMessagePermissionsAsync = async (room, { uid, username,
 	if ((room.muted || []).includes(username)) {
 		throw new Error('You_have_been_muted');
 	}
-
-	console.error(room);
-	if (room.t === 'd') {
```

Comment:
Actually, I think this is necessary 'cause admins can send messages to read only rooms. And that will mean that they can send messages to deactivated users.

If that's valid, then the validation is not needed.